README.md

```diff
@@ -133,6 +133,8 @@ TARGET SELECTION:
             Filter devices by USB bus number
         --address <addr>
             Filter devices by USB device address
+        -i, --enable-reset-interface
+            Enable detection of stdio_usb-compatible reset interface for custom USB devices
```

Comment:
Only a small niggle, but IMHO `--enable-reset-interface` sounds like it's doing something "active", and different to "Enable detection of ...", if you see what I mean? :thinking: 
Perhaps `--reset-interface` is more "passive" and more in line with the other `--bus`, `--address` and `--force` options?

Also, I wonder if it _might_ be worth adding more of an explanation (elsewhere in this README) of what "detection of stdio_usb-compatible reset interface" means? :man_shrugging:

Reply:
These options all do something active, I want to make it clear this adds support for the reset interface. How about `--support-reset-interface` or `--allow-reset-interface`? I feel like just `--reset-interface` is not clear enough.

Reply:
> I feel like just `--reset-interface` is not clear enough.

True, I hadn't considered that `--reset-interface` could itself also be misinterpreted as an "active" thing :laughing: 

> How about `--support-reset-interface` or `--allow-reset-interface`?

Yeah they sound like decent options. The commit-comment says "Add CLI option to enable detection of devices with reset interface", so perhaps `--detect-reset-interfaces` is another alternative? :man_shrugging: